test/integration/narrow/routes/endemics/biosecurity.test.js

```diff
@@ -37,7 +37,7 @@ describe('Biosecurity test when Optional PI Hunt is OFF', () => {
     await server.initialize()
     raiseInvalidDataEvent.mockImplementation(() => { })
     setEndemicsClaim.mockImplementation(() => { })
-    setEndemicsAndOptionalPIHunt({ endemicsEnabled: true, optionalPIHuntEnabled: false })
+    setOptionalPIHunt({ endemicsEnabled: true, optionalPIHuntEnabled: false })
```

Comment:
`endemicsEnabled: true` is still being passed here. Probably worth doing a repo wide search on the word endemics to find all the stragglers.